srcpkgs/sile/template

```diff
@@ -1,27 +1,40 @@
 # Template file for 'sile'
 pkgname=sile
-version=0.9.5.1
-revision=9
+version=0.15.6
+revision=1
 build_style=gnu-configure
-hostmakedepends="pkg-config"
+build_helper=rust
+configure_args="--with-system-lua-sources --with-system-luarocks"
+hostmakedepends="jq poppler cargo rust pkg-config"
 makedepends="harfbuzz-devel lua51-devel lua51-lpeg lua51-luaexpat
- lua51-zlib lua51-luafilesystem lua51-luasocket lua51-luasec"
-depends="lua51-lpeg lua51-luaexpat lua51-zlib lua51-luafilesystem
- lua51-luasocket lua51-luasec"
+ lua51-zlib lua51-luafilesystem lua51-luasocket lua51-luasec lua51-cassowary
+ lua51-compat53 lua51-cldr lua51-fluent lua51-loadkit lua51-linenoise
+ lua51-cliargs lua51-luarepl lua51-luasocket lua51-luasec lua51-luautf8
+ lua51-vstruct fontconfig-devel LuaJIT-devel font-sil-gentium rust-std"
+depends="LuaJIT lua51-lpeg lua51-luaexpat lua51-zlib lua51-luafilesystem lua51-cassowary
+ lua51-luasocket lua51-luasec lua51-compat53 lua51-cldr lua51-fluent lua51-loadkit
+ lua51-linenoise lua51-cliargs lua51-luarepl lua51-luasocket lua51-luasec lua51-luautf8
+ lua51-vstruct font-sil-gentium"
 short_desc="Modern typesetting system inspired by TeX"
 maintainer="John <me@johnnynator.dev>"
 license="MIT"
-homepage="http://www.sile-typesetter.org/"
-distfiles="https://github.com/simoncozens/sile/releases/download/v${version}/sile-${version}.tar.bz2"
-checksum=60cdcc4509971973feab352dfc1a86217cc1fdb12d56823f04d863afef92003a
+homepage="https://www.sile-typesetter.org/"
+distfiles="https://github.com/sile-typesetter/sile/releases/download/v${version}/sile-${version}.tar.zst"
+checksum=0ad3efc5ba6adbfab0b8034fa7d5c32501e5c486c589a35926f619794c7fc321
 
 if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" lua51 lua51-devel lua51-lpeg luaexpat lua51-zlib luafilesystem
-	 luasocket lua51-luasec"
+	hostmakedepends+=" ${makedepends}"
 fi
 
 post_install() {
-	vlicense LICENSE
+	vlicense LICENSE.md
+}
+
+post_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		local line_no=$(grep "\$(INSTALL) \$< $@" Makefile --line-number | cut -d':' -f1)
+		vsed -i "${line_no}s|$<|target/*linux*/release/\$@|" Makefile.in
```

Comment:
This is messing you up. It was designed to fix one instance of the Makefile looking in the Rust `target/release` dir, but as of v0.15.6 there are two instances of such install commands. You are patching the new second one using code meant to match the first one and hence cross wiring what is being looked for when.

I suggest dropping this hack entirely and using whatever Void's equivalent of Arch's `prepare()` phase (i.e. runs before configure, not after) to cherry pick 19c7c1d4. That should fix the problem properly. You should be able to download that patch as a source and apply it before configuring.